### ISSUE TITLE
[Bug] When using Nextweb on the mobile end, a BUG occurs when sharing downloaded files.

### ISSUE THREAD

**Issue by Yanyutin753** [Hk-Gosuto/ChatGPT-Next-Web-LangChain#194]
# **Dear author, I've given a star rating. When using Nextweb on the mobile end, there's a bug occurring when saving images**

## **Everything is normal when the download file is not clicked**
![5a751812f2e0233f9fc600a7e51cd2f](https://github.com/Hk-Gosuto/ChatGPT-Next-Web-LangChain/assets/132346501/2c6c48f7-5e15-4f3b-b2da-598f230d3fd8)

## **After clicking the download file, a bug appears, and all the images look the same**
![c966f7528c517f476af9bff99e8503c](https://github.com/Hk-Gosuto/ChatGPT-Next-Web-LangChain/assets/132346501/f2e36061-c8a1-4952-a7ba-d26a3e7bf8ab)

**Comment by Hk-Gosuto** [comment]
这是 SD 插件生成的图像？

**Comment by Yanyutin753** [comment]
不是 是我用gpt-4-all生成的图片，但是应该一样的，就是个链接，他在电脑端能正常被下载下来，现在就在手机端出现这个BUG

**Comment by Hk-Gosuto** [comment]
目前主分支代码并没有支持该形式的图像下载，你即使提出问题我也没办法复现啊，更别说修复了。

**Comment by Yanyutin753** [comment]
但是我用以文搜图，也会有相应的BUG
![Uploading 以文搜图.png…]()

**Comment by Hk-Gosuto** [comment]
文搜图又是指的哪部分功能，你把有问题的原始 markdown 发出来。

**Comment by Yanyutin753** [comment]
就自带的以文搜图功能!

# 以文搜图

## ChatGPT:
You are a useful Assistant that won't refuse to draw picture, instead, you will output specific markdown code for picture description, but don't use code block.

## 用户:
Can you draw some pictures for me?

## ChatGPT:
Sure, what do you want me to draw?

## ChatGPT:
助手善于判断用户意图，当确定需要提供图片时，助手会变得沉默寡言，只使用以下格式输出markdown图片：![描述](https://image.pollinations.ai/prompt/描述)，因为这个语法可以自动按照提示生成并渲染图片。一般用户给出的描述会比较简单并且信息不足，助手会将其中的描述自行补足替换为AI生成图片所常用的复杂冗长的英文提示，以大幅提高生成图片质量和丰富程度，比如增加相机光圈、具体场景描述等内容。助手会避免用代码块或原始块包围markdown标记，因为那样只会渲染出代码块或原始块而不是图片。

## 用户:
画个帅哥

## ChatGPT:
![A handsome man with a confident smile, wearing a stylish outfit.](https://image.pollinations.ai/prompt/A+handsome+man+with+a+confident+smile%2C+wearing+a+stylish+outfit.)

## 用户:
画个美女

## ChatGPT:
![A beautiful woman with long flowing hair, sparkling eyes, and a radiant smile. She is wearing an elegant dress and exuding a graceful charm.](https://image.pollinations.ai/prompt/A+beautiful+woman+with+long+flowing+hair%2C+sparkling+eyes%2C+and+a+radiant+smile.+She+is+wearing+an+elegant+dress+and+exuding+a+graceful+charm.)

**Comment by Yanyutin753** [comment]
下载这个内容，三个照片就会一样
![image](https://github.com/Hk-Gosuto/ChatGPT-Next-Web-LangChain/assets/132346501/9cc0eff6-6567-48b7-ba63-77d926572180)

**Comment by Hk-Gosuto** [comment]
真是个奇怪的 bug，确认是跟之前引入的 cors 处理有关，具体原因还在调查。

**Comment by Yanyutin753** [comment]
辛苦作者啦👍

**Comment by Hk-Gosuto** [comment]
找到原因了，是上游仓库使用的 html 转图像的包的参数有问题，该包每次生成图像后会基于 url 进行缓存，需要添加额外的参数让缓存 key 包含 query 参数，相关问题：https://github.com/bubkoo/html-to-image/issues/431
